base/Dockerfile.ubi8

```diff
@@ -67,6 +67,11 @@ RUN microdnf install yum \
 ADD --chown=appuser:appuser target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/share/doc/* /usr/share/doc/${ARTIFACT_ID}/
 ADD --chown=appuser:appuser target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/share/java/${ARTIFACT_ID}/* /usr/share/java/${ARTIFACT_ID}/
 
+# Operator dependencies for metrics integration
+RUN mkdir -p /opt/javaagents
+ADD --chown=appuser:appuser target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/share/java/${ARTIFACT_ID}/jolokia-jvm-*-agent.jar /opt/javaagents/.
```

Comment:
@jhlee8 on line 68 i think we are also copying these jars to another folder. can they just be referenced from there? it probably doesn't cause any issues having them in both places but it does increase the image size a little. if possible can we avoid that?

Reply:
Yeah I think this is a double copy good find.

> Please let me know if we should place them in /opt/caas/lib or anywhere else for compatibility reasons.

I think somewhere under /usr/share/java is the ideal place in terms of "GNU Linux best practices", stuff that goes to /opt should be entirely standalone and run-able.

Reply:
Removed the redundant copying to `/opt/javaagents` since the jars are already copied to `/usr/share/java`.

@xnulinu If there is preference to have jolokia/prometheus jars referenced/placed in certain ways, please let me know.

Reply:
@jhlee8 `/user/share/java` also works given we already dump everything there.